app/core/__init__.py

```
@@ -0,0 +1,100 @@
+    "TransactionType",
+    "PersonType",
+    "PersonRole",
+    "CommitteeRole",
+    "State",
+    # Database
+    "db_manager",
+    "UnifiedDatabaseManager",
+    # Loaders
+    "UnifiedStateLoader",
+    "UnifiedStateProcessor",
+]
```

Ruff (0.14.14)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)